sql/parser/sql.y

```diff
@@ -3576,6 +3592,7 @@ reserved_keyword:
 | CASE
 | CAST
 | CHECK
+| CLOCK_TIMESTAMP
```

Comment:
These are not reserved keywords in the SQL sense. I think you should be adding these "keywords" to the `type_func_name_keyword` list.